analyses/create-subset-files/create_subset_files.sh

```diff
@@ -30,13 +30,30 @@ cd "$script_directory" || exit
 
 # directories that hold the full files for the release and the subset files
 # generated via these scripts
+DATA_DIRECTORY=../../data
 FULL_DIRECTORY=../../data/$RELEASE
 SUBSET_DIRECTORY=../../data/testing/$RELEASE
 # If run subsetting only, we need to make this directory
 if [ ! -d ${SUBSET_DIRECTORY} ]; then
     mkdir -p ${SUBSET_DIRECTORY}
 fi
 
+# download Illumina methylation annotations file if does not exist in data
```

Comment:
needed to add this since it needs to go in the testing directory and is not downloaded by default